framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/widget/WidgetHeader.tsx

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import * as React from 'react';
+import styles from './Widget.module.scss';
+
+const WidgetHeader: React.FC = props => <div className={styles.header}>{props.children}</div>;
```

Comment:
widget header is `h2` so u can use `Text` `h2` here

Reply:
`<Text>`  or any other component should be used by the user (one who implement any specific widget) himself. e.g. 

```
<Widget>
   <WidgetHeader>
      <Text name={'h2'}>My header</Text>
      <Icon name={'info'}>
``

Reply:
then the `header` style is worng? it's specifying the font of the child